index.js

```diff
@@ -0,0 +1,15 @@
+const dirTree = require('directory-tree')
+const fs = require('fs')
+const fileName = 'file-index.json'
+const minified = true
+
+function dirCallback (item) {
+  delete item.path // remove 'path' key from all directory objects
+}
+
+const tree = dirTree('nursery_catalogs', { attributes: ['type'] }, null, dirCallback)
+
+fs.appendFile(fileName, JSON.stringify(tree, null, minified ? 0 : 2), function (err) {
+  if (err) throw err
```

Comment:
Personally, I think our general code style is to avoid this style of `if` statement and expand that out to:

```
if (err) {
  throw err;
}
```
  
  Not the end of the world to keep it a one-liner, I know it's shorter, the longer version is maybe a bit more clear.

Reply:
Yes I agree, clarity > code length